### PR TITLE
fix the error msg of func TestParseRunVolumes

### PR DESCRIPTION
**Pull request by xlgao-zju**
fix the error msg of func `TestParseRunVolumes` according to #16013

Signed-off-by: xlgao-zju xlgao@zju.edu.cn

### CODE REVIEW

File: runconfig/parse_test.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func TestParseRunVolumes(t *testing.T) {
 	}
 
 	if _, hostConfig := mustParse(t, "-v /containerTmp:ro -v /containerVar:rw"); hostConfig.Binds == nil || compareRandomizedStrings(hostConfig.Binds[0], hostConfig.Binds[1], "/containerTmp:ro", "/containerVar:rw") != nil {
-		t.Fatalf("Error parsing volume flags, `-v /hostTmp:/containerTmp:ro -v /hostVar:/containerVar:rw` should mount-bind /hostTmp into /containeTmp and /hostVar into /hostContainer. Received %v", hostConfig.Binds)
+		t.Fatalf("Error parsing volume flags, `-v /containerTmp:ro -v /containerVar:rw` should mount-bind /containeTmp into /ro and /containerVar into /rw. Received %v", hostConfig.Binds)
```
Comment (duglin) [inline runconfig/parse_test.go]: s/containeTmp/containerTmp

**Comment by duglin** [comment]
minor typo, but once fixed, LGTM

**Comment by xlgao-zju** [comment]
updated
@duglin

**Comment by vdemeester** [comment]
LGTM 🐻

**Pull request merged**